packages/trpc/server/routers/loggedInViewer/outOfOffice.handler.ts

```diff
@@ -134,12 +134,14 @@ export const outOfOfficeCreate = async ({ ctx, input }: TBookingRedirect) => {
   if (existingOutOfOfficeEntry) {
     throw new TRPCError({ code: "BAD_REQUEST", message: "booking_redirect_infinite_not_allowed" });
   }
+  const startDateUtc = dayjs.utc(startDate).add(input.offset, "minute");
```

Comment:
The date form the `DateRangePicker` returns the date in utc but with offset hours from the user's browser timezone. 

So in NY `startDate` from `DateRangePicker` would be 05:00 instead of 00:00, wich ended up saving wrong days in database